core/src/App.html

```diff
@@ -115,8 +115,10 @@
     if (data.params.preserveView) {
       const pv = component.get().preservedViews;
       const nextPath = buildPath(component, data.params);
+      const pathUrlRaw = component.get().pathUrlRaw;
```

Comment:
The solution works well 👍 
I can see you're calling `component.get()` multiple times. Wouldn't it be better to use 
```javascript
const {pathUrlRaw, preservedViews, currentNode} = component.get();
```
I'm not sure but there might be also a performance advantage for this approach.

Reply:
I like this suggestion!